caffe2/python/tutorials/MNIST.ipynb

```diff
@@ -553,6 +553,10 @@
     "workspace.RunNetOnce(train_model.param_init_net)\n",
     "# creating the network\n",
     "workspace.CreateNet(train_model.net, overwrite=True)\n",
+    "# Also initialize the test model before training\n",
+    "workspace.RunNetOnce(test_model.param_init_net)\n",
```

Comment:
Here is my solution:
`test_model = model_helper.ModelHelper(name="mnist_test", arg_scope=arg_scope, init_params=False)
data, label = AddInput(test_model, batch_size=100,db=os.path.join(data_folder, 'mnist-test-nchw-leveldb'),db_type='leveldb')

workspace.RunNetOnce(test_model.param_init_net)

softmax = AddLeNetModel(test_model, data)
AddAccuracy(test_model, softmax, label)

workspace.CreateNet(test_model.net)
workspace.RunNet(test_model.net.Proto().name)`


First, just define the test_model's input part, and immediately initalize it.
Then add model to it.
At last, create and run.